docs/release-notes/5.7.44-53.md

```diff
@@ -0,0 +1,48 @@
+# Percona Server for MySQL 5.7.44-53 (2025-02-05)
+
+--8<--- "eol.release.md"
+
+## Release Information  
+
+Percona Server for MySQL 5.7.44-53 builds on MySQL 5.7.44 Community Edition,  
+adding enterprise-grade features and security enhancements from Percona.  
+
+## Bug fixes
+
+* [Bug#36234681](https://github.com/mysql/mysql-server/commit/a8c96ce81ab): InnoDB: The server encounters a failed assertion where the result does not equal
+`FTS_INVALID`. This indicates an unexpected state in the code, requiring
+attention to prevent further errors.
+
+* [Bug#37189985](https://github.com/mysql/mysql-server/commit/0fe52ea5c8f): Using `INPLACE ALTER TABLE` results in server exits and corrupts spatial
+indexes. This issue affects data integrity and must be fixed to prevent
+similar failures in future operations.
+
+* [Bug#33691659](https://github.com/mysql/mysql-server/commit/551288baa7a): When performing `ON DELETE CASCADE` with a generated column, the server exits
+in `innobase_get_computed_value`.
+
+* [Bug#33327093](https://github.com/mysql/mysql-server/commit/0ac176453bf): A foreign key (FK) constraint causes an assertion failure in
+`row_MySQL_pad_col`, leading to a server exit.
+
+* [Bug#34574604](https://github.com/mysql/mysql-server/commit/632ca5cfe1b): The server experienced an exit due to `ERROR 1712 (HY000): Index PRIMARY was corrupted`.
+
+* [Bug#36593235](https://github.com/mysql/mysql-server/commit/9a32e4495c5): "The MySQL server version 8.3.0 experienced a server exit at `Item_rollup_sum_switcher::current_arg`.
+
+* [Bug#37068527](https://github.com/mysql/mysql-server/commit/04b8be52e59): When they compiled with XCode 16, a bad comparator caused a server exit during the mysqld initialization. 
+
+* [Bug#37132323](https://github.com/mysql/mysql-server/commit/dd8b0c5c7d6): The server exited because the database cache wasn't flushed when a user was dropped.
+
+* [Bug#37063288](https://github.com/mysql/mysql-server/commit/b76c48b031d): The server exited because AppArmor denied access to the files `/proc/self/task/<tid>/mem`.
+
+* [Bug#36615714](https://github.com/mysql/mysql-server/commit/6e08d4427fc): "The server exited because of a buffer overrun in my_print_help.
+
+* [Bug#30875669](https://github.com/mysql/mysql-server/commit/c510fe1470a): The server exited because the core client couldn't send a query containing a number sign or double dash in a hint comment.
+
+* [Bug#36879147](https://github.com/mysql/mysql-server/commit/a6b6a6b9861%7Chttps://g): "The server exited because the InnoDB FULLTEXT index limited `FTS_DOC_ID` to the maximum unsigned 32-bit value.
```

Comment:
```suggestion
* [Bug#36879147](https://github.com/mysql/mysql-server/commit/a6b6a6b9861%7Chttps://g): The server exited because the InnoDB FULLTEXT index limited `FTS_DOC_ID` to the maximum unsigned 32-bit value.
```